views/login.handlebars

```diff
@@ -0,0 +1,36 @@
+<!-- views/login.handlebars -->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login - Restaurant POS</title>
+    {{!-- <link rel="stylesheet" type="text/css" href="styles.css"> --}}
+</head>
+<body>
+    <header>
+        <h1>Restro POS</h1>
+		<br>
+    </header>
+    <main>
+        <section class="login-container">
+            <h2>User Login</h2>
+            <form method="POST" action="/login">
```

Comment:
please remove the method and action as the javascript will handle form submission